### DISCUSSION TITLE
Meetings/maps

### DISCUSSION THREAD

**Discussion by ErikOla** [gityopie/odoo-addons#163] [Q&A]
Is it possible to use web_google_maps and have a view on the meetings module so that you can see where meetings are in, for example, a certain week?

**Comment by gityopie** [comment]
Hi @ErikOla 
Yes I think it is possible.
Which Odoo version you want to have it?

**Reply by ErikOla** [reply]
Hi @gityopie,  v14 CE. That would be great. Meetings have a location field but are not always used. Would you use the meeting location field or can you use the adres of the linked opportunity, or opportunity/contact?

**Reply by gityopie** [reply]
@ErikOla please check this video https://drive.google.com/file/d/1R6YYk9swUh1kQTO2UUj2QIfieF-18yaN/view?usp=sharing
it is using existing field "Location" and when you create new meeting from either CRM or Contact, it will take partner related address and geolocation data.

**Reply by gityopie** [reply]
@ErikOla if you want to try it on your test/dev instance, you can find the module `calendar_location` on this working [branch](https://github.com/gityopie/odoo-addons/tree/14.0-meeting-maps)
